handshake_messages.go

```diff
@@ -1407,10 +1407,16 @@ func (m *certificateMsg) unmarshal(data []byte) alert {
 	return alertSuccess
 }
 
+type certificateEntry struct {
+	data       []byte
+	ocspStaple []byte
+	sctList    []byte
```

Comment:
Since this is implemented, can you add SignedCertificateTimestamps support above while you are at it?